modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/CommitWriteInvokeClosure.java

```diff
@@ -84,10 +103,10 @@ public OperationType operationType() {
 
     @Override
     public void onUpdate() {
-        assert operationType == OperationType.PUT ? updateTimestampLink != null : updateTimestampLink == null :
+        assert operationType == OperationType.PUT ? true : updateTimestampLink == NULL_LINK :
```

Comment:
Oh my god!
You know that `a ? true : b` is the same as `a || b`?

Reply:
Fix it